blaze/tests/test_interactive.py

```diff
@@ -201,6 +201,19 @@ def test_table_resource():
         assert isinstance(t.data, CSV)
         assert into(list, compute(t)) == into(list, csv)
 
+def test_explicit_dshape():
+    ds = dshape("var * {a: ?float64, b: ?string, c: ?float32}")
```

Comment:
can you put a comment saying that if this isn't overridden it will return `c: int64` instead. This will help debug this later if it ever changes.